src/shared/components/mutationTable/column/PanelColumnFormatter.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+	Mutation,
+	DiscreteCopyNumberData
+} from 'shared/api/generated/CBioPortalAPI';
+
+interface PanelColumnFormatterProps {
+	data: Mutation[] | DiscreteCopyNumberData[];
+	sampleToMutationGenePanelId: {[sampleId:string]: string} | undefined;
+}
+
+class PanelColumn extends React.Component<PanelColumnFormatterProps, {}> {
+	constructor(props: PanelColumnFormatterProps) {
+		super(props);
+	}
+
+	render() {
+    const { data, sampleToMutationGenePanelId } = this.props;
```

Comment:
seems like there's some indentation issues here.   you can run prettier.js on this file.   see command in scripts of package.json